src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -212,39 +212,43 @@ static ucs_status_t uct_cuda_copy_md_set_ctx(uct_cuda_copy_md_t *md)
     CUcontext ctx;
 
     ucs_assertv(!md->cuda_ctx_retained,
-                "cuda context has been already retain for the md %p", md);
+                "md %p: cuda context has already been retained", md);
 
     status = UCT_CUDADRV_FUNC_LOG_ERR(cuDeviceGet(&dev, 0));
     if (status != UCS_OK) {
-        goto ret;
+        goto err;
     }
 
     status = UCT_CUDADRV_FUNC_LOG_ERR(cuDevicePrimaryCtxRetain(&ctx, dev));
     if (status != UCS_OK) {
-        goto ret;
+        goto err;
     }
 
     status = UCT_CUDADRV_FUNC_LOG_ERR(cuCtxSetCurrent(ctx));
     if (status != UCS_OK) {
-        goto release_ctx;
+        goto err_release;
     }
 
     md->cuda_ctx_retained = 1;
+    ucs_debug("retained the primary context on the gpu %d and made it current "
```

Comment:
1. IMO the message is too long.
2. Maybe include the `md` as well?
How about:
`md %p: cuda context retained on gpu %d`

Reply:
"md %p: cuda primary context retained on gpu %d"

Reply:
reduced the message.
I left the word 'primary' because this is a special context. And made debug messages identical:
```
ucs_debug("md %p: retained the primary context on the gpu %d", md, dev);
ucs_debug("md %p: released the primary context on the gpu %d", md, dev);
```

Reply:
```
ucs_assertv(!md->cuda_ctx_retained,
                "md %p: cuda primary context has already been retained", md);
ucs_debug("md %p: cuda primary context retained on gpu %d", md, dev);
ucs_debug("md %p: cuda primary context released on gpu %d", md, dev);
 ```